Dockerfile

```diff
@@ -43,7 +45,7 @@ RUN curl -SLO "${NODE_LOCATION}" \
 
 COPY package*.json ./
 
-RUN npm ci
+RUN npm ci --build-from-source --sqlite=/usr/lib
```

Comment:
Is it possible to only build sqlite from source? There might be other modules you're now unnecessarily building from source and avoiding those would make sense. In fact maybe you can just omit the `--build-from-source` altogether since if we can use the prebuilt sqlite that should be fine?